### ISSUE TITLE
Having trouble to edit web.config file under system.web

### ISSUE THREAD

**Issue by fadimaster** [PowerShell/xWebAdministration#355]
Hi there,

is it possible to add in web.config file the following line under system.web:

```
<system.web>
<httpCookies requireSSL="true" />
<system.web /> 
```

**Comment by johlju** [comment]
I don't think that is possible with any of to today's resources. Please look if PR #330 will solve you rproblem once merged.

**Comment by fadimaster** [comment]
i'me trying to use the xWebConfigProperty instead of xWebConfigKeyValue and getting the error: "undefined DSC resource" on powershell means that Xwebadministration Module doesn't recognize xWebConfigProperty  parameter

**Comment by johlju** [comment]
xWebConfigProperty is a proposed (new) resource in pull request #330. It has not been merged into the resource module yet. If you want to test it out before it is merged into the resource module (and eventually released), you can check out the working branch of the PR author.

**Comment by regedit32** [comment]
This is now resolved by PR #330.  Please re-open if you feel this needs to be further addressed.
